design/ceph/object-store-zonegroup.md

```diff
@@ -0,0 +1,128 @@
+
+Multisite also allows object stores to be independent and isloated from other object stores in a cluster.
+
+For more information on Ceph multsite, please read the [object-store-realm](/design/ceph/object-store-realm.md) design document or [Ceph's documentation](http://docs.ceph.com/docs/master/radosgw/multisite/) on the feature.
+
+## Object Store Zonegroup Walkthrough
+
+The resource descibed in this design document represents the zonegroup in the Ceph Multisite data model.
+
+### Creating an Object Store Zonegroup
+
+#### Config
+
+When the storage admin is ready to create a multisite zonegroup for object storage, the admin will name the zonegroup in the metadata section on the configuration file.
+
+In the config, the admin must configure the realm the zonegroup is in and whether or not the zonegroup is the master zonegroup for the realm.
+
+The first object-store-zonegroup resource in a realm must be a master zonegroup. This means `isMaster` must be set to `True`.
+
+This example `object-store-zonegroup.yaml`, names a zonegroup `my-zonegroup`.
+```yaml
+apiVersion: ceph.rook.io/v1alpha1
+kind: CephObjectStorezonegroup
+metadata:
+  name: zonegroup-a
+  namespace: rook-ceph
+spec:
+  realm: my-realm
+  isMaster: True
+```
+
+Now create the object store zonegroup.
+```bash
+kubectl create -f object-store-zonegroup.yaml
+```
+
+#### Steps
+
+1. At this point the Rook operator recognizes that a new object store zonegroup resource needs to be configured. The operator will start creating the resource to start the object store zonegroup.
+
+2. After these steps the admin should start up a master [object-store-zone](/design/ceph/object-store-zone.md) for the zonegroup, and an [object-store](/design/ceph/object-store.md) in the master object-store-zone.
+
+3. Once all of the resources in #2 are created, the operator will create a zonegroup on the rook Ceph cluster and the object-store-zonegroup resource will be running.
+
+#### Notes
+
+1. The realm named in the `realm` section must be the same as the object-store-realm resource the zonegroup is a part of.
+2. If there is already a master object-store-zonegroup and another is created as a master zonegroup for the same realm, the 2nd object-store-zonegroup resource will error and not be created.
+3. When resource is deleted, zonegroup are not deleted from the cluster. Zonegroup deletion must be done through toolboxes.
+
+### Creating an object-store-zonegroup when syncing from another Ceph Cluster
+
+When the storage admin is ready to sync data from another Ceph cluster with multisite set up (primary cluster) to a Rook Ceph cluster (pulling cluster), the pulling cluster needs to have information about the primary cluster's realm and zonegroup.
+
+This is accomplished by creating object-store-realm and object-store-zonegroup resources with the same name as the realm and zonegroup of the primary cluster.
```

Comment:
why do you need a zonegroup as well? according to this: https://github.com/rook/rook/pull/4520/files#diff-00cb90a9a9a7d7efb7e118b1159e8c74R151 the information is pulled at realm level